spidermon/contrib/actions/slack/__init__.py

```diff
@@ -110,14 +116,22 @@ def _send_user_message(
     def _send_channel_message(
         self, channel, text, parse="full", link_names=1, attachments=None
     ):
-        self._client.chat_postMessage(
+        result = self._client.chat_postMessage(
             channel=channel,
             text=text,
             parse=parse,
             link_names=link_names,
             attachments=self._parse_attachments(attachments),
             username=self.sender_name,
             icon_url=self._get_icon_url(),
+            thread_ts=self.thread_id,
+            # thread_ts = "1659182317.958909",
+            reply_broadcast=self.also_send_channel,
+            mrkdwn=self.mrkdwn
+        )
+
+        self.thread_id = (
```

Comment:
**issue:** if we reuse the `thread_id` all the Slack messages will be grouped under the same thread. This is a nice feature to have in certain projects, but it should probably be implemented separately and not as the default behavior, or it'd break backward compatibility.

Reply:
The main reason of this issue was exactly to make it possible to send messages to the same thread (using `thread_ts` parameter). However, if we don't provide a thread_id, we shouldn't reuse the same for all messages.

Reply:
You are right, should have read the ticket description 😅 Makes sense to rely on the provided `thread_id`, and leave it to users to define it.